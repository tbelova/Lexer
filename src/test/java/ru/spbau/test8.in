sum2 (a, b, mod) {
	return (a + b) % m; //sum of two
}
