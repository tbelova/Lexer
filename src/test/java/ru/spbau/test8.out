PROGRAM((1,0)(4,4))
   STATEMENTS((1,0)(3,0))
      FUNCTION((1,0)(3,0))
         IDENT((1,0)(1,3))[sum2]
         IDENTS((1,6)(1,14))
            IDENT((1,6)(1,6))[a]
            IDENTS((1,9)(1,14))
               IDENT((1,9)(1,9))[b]
               IDENTS((1,12)(1,14))
                  IDENT((1,12)(1,14))[mod]
         BLOCK((1,17)(3,0))
            STATEMENTS((2,1)(2,19))
               STATEMENT((2,1)(2,19))
                  RETURN((2,1)(2,18))
                     EXPRESSION((2,8)(2,18))
                        REM((2,8)(2,18))
                           SUM((2,9)(2,13))
                              IDENT((2,9)(2,9))[a]
                              IDENT((2,13)(2,13))[b]
                           IDENT((2,18)(2,18))[m]
